npm/react/examples/sass-and-ts/package.json

```diff
@@ -13,6 +13,6 @@
   },
   "devDependencies": {
     "@cypress/react": "file:../../dist",
-    "node-sass": "4.14.1"
+    "sass": "1.32.0"
```

Comment:
what is the difference between these two? guessing this one is implemented in something other than js?

Reply:
The official sass release uses Dart and can be installed via npm, and Nodesass uses libsass, which is the (older) C version.

Reply:
Check out this blog post: https://sass-lang.com/blog/libsass-is-deprecated